### ISSUE TITLE
make _runConfigCallbacks() runnable only once

### ISSUE THREAD

**Issue by imonteiro** [zfdatagrid/grid#368]
_Original author: martin.m...@gmail.com (January 10, 2011 00:05:33)_

_runConfigCallbacks should internaly check if it was not executed already
at this moment setAjax() and deploy() call it twice

_Original issue: http://code.google.com/p/zfdatagrid/issues/detail?id=543_

**Comment by imonteiro** [comment]
_From martin.m...@gmail.com on January 10, 2011 01:38:53_
_runConfigCallbacks () should be public method

**Comment by imonteiro** [comment]
_From bento.vi...@gmail.com on January 10, 2011 03:20:02_
Hi Martin,

runConfigCallbacks was created by you.

And as you stated in method declaration

```
 * @param array|array of array $classCallbacks key should be lowercase, functions to call once before deploy() and ajax() functions
```

This is only used in jQgrid. You can change this at you will. 

Best Regards,
Bento Vilas Boas

**Comment by imonteiro** [comment]
_From martin.m...@gmail.com on January 10, 2011 07:58:41_
fixed in #1549
